core/src/main/java/org/wso2/carbon/kernel/internal/DataHolder.java

```diff
@@ -79,7 +79,7 @@ public void setRuntimeManager(RuntimeManager runtimeManager) {
     /**
      * Setter method of ${@link ConfigResolver}.
      *
-     * @param optConfigResolver
+     * @param optConfigResolver configuration resolver object
```

Comment:
There are pending javadoc in this class. This is being handled in a separate PR, therefore shall we revert changes in this file